### PR TITLE
✨ Add KF DRC phenotype Mapper

### PR DESCRIPTION
**Pull request by liberaliscomputing**
This PR adds a Python mapper for KF DRC phenotype resources.

### CODE REVIEW

File: kf_model_fhir/mappers/resources/phenotype.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        'display': 'Negative'
+    },
+    constants.PHENOTYPE.OBSERVED.YES: {
+        'system': 'http://terminology.hl7.org/CodeSystem/v3-ObservationInterpretation',
+        'code': 'POS',
+        'display': 'Positive'
+    }
+}
+
+def yield_participants(eng, table, study_id, participants):
```
Comment (fiendish) [inline kf_model_fhir/mappers/resources/phenotype.py:30]: not participants

File: kf_model_fhir/mappers/resources/phenotype.py [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+from kf_lib_data_ingest.common import constants
+from kf_lib_data_ingest.common.concept_schema import CONCEPT
+
+from common.utils import make_select, get, make_identifier
+
+
+# ttps://www.hl7.org/fhir/valueset-observation-status.html
+status = {
+    constants.COMMON.TRUE: 'registered'
+    constants.COMMON.FALSE: 'unknwon'
```
Comment (fiendish) [inline kf_model_fhir/mappers/resources/phenotype.py:13]:
```suggestion
    constants.COMMON.FALSE: 'unknown'
```

File: kf_model_fhir/mappers/resources/phenotype.py [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+            'resourceType': 'Observation',
+            'id': make_identifier('phenotype', study_id,
+                                  participant_id, name,
+                                  observed, event_age_days),
+            'meta': {
+                'profile': [
+                    'http://fhir.kids-first.io/StructureDefinition/phenotype'
+                ]
+            },
+            'status': status[visible],
```
Comment (fiendish) [inline kf_model_fhir/mappers/resources/phenotype.py:58]: what is this?

File: kf_model_fhir/mappers/resources/phenotype.py (hunk 1 above)
Comment (liberaliscomputing) [reply kf_model_fhir/mappers/resources/phenotype.py]: Renamed:

- phenotype.py >> kfdrc_phenotype.py
- yield_participants >> yield_kfdrc_phenotype

File: kf_model_fhir/mappers/resources/phenotype.py (hunk 2 above)
Comment (liberaliscomputing) [reply kf_model_fhir/mappers/resources/phenotype.py]: deprecated that dictionary for now.

File: kf_model_fhir/mappers/resources/phenotype.py (hunk 3 above)
Comment (liberaliscomputing) [reply kf_model_fhir/mappers/resources/phenotype.py]: deprecated!

**Pull request merged**